web/js-file.js

```diff
@@ -1,6 +1,8 @@
 function funk(a,b) {
  if (a == b)
   return a;
+ if (a == d)
```

Comment:
Reporter: JSHINT
Rule: ?
Severity: WARN
File: ../../../web/js-file.js L4
 

&apos;d&apos; is not defined.: if (a == d)

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>